ui/webui/src/index.js

```diff
@@ -32,6 +32,7 @@ import cockpit from "cockpit";
  */
 import "../pkg/lib/patternfly/patternfly-4-overrides.scss";
 import "./components/app.scss";
+import "./components/AnacondaAboutModal.scss";
```

Comment:
This should be imported from the AnacondaAboutModal.jsx file instead for code readability purposes.